__tests__/main.test.ts

```diff
@@ -67,6 +67,46 @@ test('runSorald can ratchet from HEAD~', async () => {
   ]);
 });
 
+test('runSorald with ratchet does nothing when there are no violations in changed code', async () => {
+  // arrange
+  const workdir = await helpers.createTempdir();
+  const repo = await git.init(workdir);
+
+  // create a commit with a single violation each of rules 2164 (math on float) and 1854 (dead store)
+  const className = 'Main';
+  const filePath = path.join(workdir, `${className}.java`);
+  const baseMethodBody = `float a = 2;
+  float b = 2; // 1854: dead store
+  b = 3;
+  float c = a / b; // 2164: math on float
+  System.out.println(c);`;
+  const initialClassContent = wrapInClassInMainMethod(
+    className,
+    baseMethodBody
+  );
+  helpers.createFile(filePath, initialClassContent);
+  await repo.add(filePath);
+  await repo.commit('Initial commit');
+
+  // create a new commit with no violations
+  const secondCommitMethodBody = `${baseMethodBody}
+  b = c;
+  System.out.println(b);`;
+  const secondCommitClassContent = wrapInClassInMainMethod(
+    className,
+    secondCommitMethodBody
+  );
+  await helpers.createFile(filePath, secondCommitClassContent);
+  await repo.add(filePath);
+  await repo.commit('Update file');
+
+  const soraldJarUrl =
+    'https://github.com/SpoonLabs/sorald/releases/download/sorald-0.1.0/sorald-0.1.0-jar-with-dependencies.jar';
+  const repairs = main.runSorald(workdir, soraldJarUrl, 'HEAD~');
```

Comment:
Aren't you getting a timeout error here because you haven't setup `jest.timeout` for this test? You said that you will configure it globally but I am not aware if you have already done that.

Reply:
See line 20 :)

Reply:
Yeah, that works too :)